### ISSUE TITLE
"Get external IP script" not discussed

### ISSUE THREAD

**Issue by ulfmartin** [Stadicus/guides#82]
We set up a daemon which requires "getpublicip.service", but we did not implement that yet.
There is a missing part in the instructions which is about getting the public IP and saving it
to a temp file in memory. You do have those scripts in the repo, though.

**Comment by Stadicus** [comment]
Is this the part you are referencing to? Is anything missing?
https://github.com/Stadicus/guides/blob/master/raspibolt/raspibolt_40_lnd.md#public-ip-script

This is somewhat of a hack and will be removed as soon as LND brings full NAT compatibility (should be included in the next release: https://github.com/lightningnetwork/lnd/pull/1109).

**Comment by ulfmartin** [comment]
Oh, I now notice what went wrong in the guide.
I the order of steps is a little wrong here.
In the section "Bitcoin", "Autostart bitcoind" the bitcoind.service introduces "Wants=getpublicip.service", before we created that service.
The getpublicip.service and the corresponding sh script is then explained at the beginning of the "Lightning" section. So, I would say it's logical to do it the other way around or make a remark, that we cannot use the bitcoin daemon yet until the following step.

**Comment by ulfmartin** [comment]
There was one more thing, actually.
When attempting to write to /run/anyfilename, I would get a "permission denied" error, even when using sudo (e.g. sudo echo sometext > /run/somefile). However, I found that I can fix that issue by using /run/shm/anyfile. So I changed my getpublicip.sh script to output to that folder.
I do not really understand the /run directory, so someone might want to have a second look on that.
